fgd/point/prop/prop_floor_button.fgd

```diff
@@ -10,6 +10,10 @@
 		"models/props/portal_button_damaged02.mdl": "Slightly Damaged"
 		"models/props/portal_button_damaged01.mdl": "Very Damaged"
 		]
+	solid(choices) : "Collisions" : 0 : "Don't change, setting this to a value other than 0 causes the button to become non-solid." =
```

Comment:
You should be able to do this as `readonly` to prevent users from changing it. Or have another value to allow non-solidity.

```suggestion
	solid(choices) readonly : "Collisions" : 0 : "Don't change, setting this to a value other than 0 causes the button to become non-solid." =
```

Reply:
I didn't think that worked on choices-type keyvalues.